style.css

```diff
@@ -7,6 +7,84 @@
   width: auto;
 }
 
+body {
+  overflow-x: hidden;
+}
+
+/* Mobile menu  */
+
+.navmenu {
+  width: 100%;
+  font-family: 'inter', sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: left;
+  height: 0vh;
+  background-color: #3c3a39;
+  position: fixed;
+  top: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.ham-menu {
+  width: 15px;
+  position: relative;
+  top: 33px;
+  left: 8.3rem;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.navmenu {
+  width: 374px;
```

Comment:
- [ ] I would recommend not setting a width because we kind of indirectly limiting ourselves, there are mobiles that are > 374px width, for example, I used iPhone 12 pro.
Other than that great implementation and perfect UI 🤩 
![image](https://user-images.githubusercontent.com/87689547/156086181-fb1de0fe-3306-4f99-94b0-0d40dd98b099.png)